trip_sharing-frontend/src/components/Search_page/index.js

```diff
@@ -1,11 +1,37 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import './index.css'
 
 import Header from '../../containers/Header'
-import TripsList from '../../containers/Trips_list'
-
-export default () => (
-  <div>
-    <Header />
-    {/* <TripsList /> */}
-  </div>
-)
\ No newline at end of file
+import SearchList from '../../containers/SearchList'
+import { fetchSearching } from '../../utils/fetch_functions';
+
+
+class SearchPage extends Component {
+  
+  componentWillMount(){
+    this.props.dispatch(fetchSearching(this.props.match.params.searchTrip))
+  }
+
+
+  render(){
+    return(
+    <div>
+      <Header />
+      <div className = "SearchPage">
+        <h1> Search results: </h1>
+      </div>
+      <SearchList passing = {this.props.trips}/>
+    </div>
+
+    )
+  }
+}
+
+const mapStateToProps = ( { tripsReducer }) => {
+  return ({
+    trips: tripsReducer.trips
+  })
+}
```

Comment:
`const mapStateToProps =  tripsReducer  => ({
     trips: tripsReducer.trips
 })`
better use es6 in this case. try this out